### DISCUSSION TITLE
About compatibility issues on Android version 8.0

### DISCUSSION THREAD

**Discussion by liukkm** [pmndrs/zustand#2714] [Q&A]
![816cd85894cb32cba79bee6ba2b67ed1_compress](https://github.com/user-attachments/assets/dc9df44c-7210-448a-8fed-a61422280b90)
As shown in the picture, I am doing a compatibility work on an Android tablet. The nextjs+zustand I use can work normally on a computer or a later version of Android, but now there is always this error on the current version of Android 8.0, and I feel at a loss. The error message appears to be due to the use of syntax or features that are not supported by older JavaScript engines. Android 8's WebView may not support some of the newer JavaScript features, which can cause syntax errors when running JavaScript code. Can you help me fix this problem?

**Comment by dai-shi** [comment]
I wonder if CJS instead of ESM will work or not.

We need someone familiar with RN configuration. Metro config?
Can anyone help?

**Reply by dbritto-dev** [reply]
@liukkm I believe that is something related to module resolution

**Reply by liukkm** [reply]
You're right. I've solved the problem.

**Comment by 221156i** [comment]
Hello, Is this problem had been solved I kind a facing similar one ?
